src/icalendar/prop/__init__.py

```diff
@@ -1736,6 +2383,48 @@ def from_ical(ical):
         except Exception as e:
             raise ValueError(f"Expected time, got: {ical}") from e
 
+    @classmethod
+    def examples(cls) -> list[vTime]:
+        """Examples of vTime"""
+        return [cls(time(12, 30))]
+
+    from icalendar.param import VALUE
+
+    def to_jcal(self, name: str) -> list:
+        """The jcal represenation of this property according to :rfc:`7265`."""
+        value = self.dt.strftime("%H:%M:%S")
+        if self.params.is_utc():
+            value += "Z"
+        return [name, self.params.to_jcal(exclude_utc=True), self.VALUE.lower(), value]
```

Comment:
When a vTime has UTC timezone on `dt` but empty params, `is_utc()` returns True but `to_jcal` outputs without "Z" because it only checks `params.is_utc()`.

Test case:
```python
t = vTime(time(12, 30, tzinfo=timezone.utc))
t.params = Parameters()  # Clear params
t.is_utc()      # True
t.to_jcal("x")  # ['x', {}, 'time', '12:30:00'] - missing Z!
```

```suggestion
    def to_jcal(self, name: str) -> list:
        """The jcal represenation of this property according to :rfc:`7265`."""
        value = self.dt.strftime("%H:%M:%S")
        if self.is_utc():
            value += "Z"
        return [name, self.params.to_jcal(exclude_utc=True), self.VALUE.lower(), value]
```